src/lwc/geFormField/geFormField.js

```diff
@@ -46,12 +46,30 @@ export default class GeFormField extends LightningElement {
     }
 
     connectedCallback() {
-        const { defaultValue } = this.element;
-        if(defaultValue) {
+        const { defaultValue, recordValue } = this.element;
```

Comment:
@kenlewis-sfdo it looks like this may have some discrepancy with your most recent work for batch row update - is there a straightforward way to combine these updates?

Reply:
This has been resolved between Kiet and Ben